GSA-TTS/FAC
# Dependency Updates: 2024/06/04

# Dependency Updates: 2024/06/04

## Changes:
Dependency update commits are cherry-picked from dependabot branches. USWDS 3.8.0 requires two new images to be added to our static image directory. 

By order of application:
https://github.com/GSA-TTS/FAC/pull/3861 - Bump esbuild-sass-plugin from 3.0.0 to 3.3.0 in /backend
https://github.com/GSA-TTS/FAC/pull/3880 - Bump aquasecurity/trivy-action from 0.19.0 to 0.21.0
https://github.com/GSA-TTS/FAC/pull/3898 - Bump eslint-plugin-cypress from 2.15.1 to 3.3.0 in /backend
https://github.com/GSA-TTS/FAC/pull/3921 - Bump glob from 10.3.12 to 10.4.1 in /backend
https://github.com/GSA-TTS/FAC/pull/3922 - Bump eslint from 8.57.0 to 9.4.0 in /backend
https://github.com/GSA-TTS/FAC/pull/3923 - Bump @uswds/uswds from 3.7.0 to 3.8.1 in /backend
    * Includes a second commit to bring over the necessary images - required for the static compiler (esbuild, build-assets.js) to complete

## How to test:
1. Switch to this branch
2. Bring up a fresh stack, regardless of how annoying it may be
    * Via a something akin to `make docker-clean | make docker-first-run | docker compose up`
    * You can probably also just nuke the node_modules, compiled, and staticfiles directories - but I'm not 100% sure on that
4. Verify that things load up properly, and that the app runs properly

## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
